waspc/data/Generator/templates/server/package.json

```diff
@@ -3,9 +3,11 @@
   "name": "server",
   "version": "0.0.0",
   "private": true,
-  "type": "module",
   "scripts": {
-    "start": "nodemon -r dotenv/config ./src/server.js",
+    "build": "npx tsc",
+    "start": "NODE_PATH=dist node dist/server.js",
+    "build-and-start": "npm run build && npm run start",
+    "start-nodemon": "nodemon -r dotenv/config --exec 'npm run build-and-start || exit 1'",
```

Comment:
Watches JavaScript and TypeScript files, and recompiles and runs the code on each change.

The `|| exit 1` part tells nodemon to continue listening for changes even if the server dies with a fatal error.

Reply:
Cool! Could you add a comment for that in the package.json itself? I think it is a good rule that if you have to explain code in the comment in the PR, then that comment better be in the code itself :D.